.github/workflows/snapshot-build.yml

```diff
@@ -172,6 +178,12 @@ jobs:
       - name: Checkout
         uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
 
+      # Keep this in sync with the fallback command from the substrate build script:
+      # https://github.com/autonomys/polkadot-sdk/blob/e831132867930ca90a7088c7246301ab29f015ba/substrate/utils/build-script-utils/src/version.rs#L28
+      - name: Get short SHA
+        run: |
+          echo SUBSTRATE_CLI_GIT_COMMIT_HASH=$(git rev-parse --short=11 HEAD) >> "$GITHUB_ENV"
```

Comment:
Why is this needed to be set explicitly? It should be doing the same thing internally, it is only different in the container build (which is Linux-specific) due to `.git` not included in image build context.